docs/pages/enterprise/hsm.mdx

```diff
@@ -0,0 +1,205 @@
+---
```

Comment:
Please consult the meta docs on docs to follow the style guide:

https://goteleport.com/docs/docs/